### DISCUSSION TITLE
是否能够取消蔡徐坤元素？！

### DISCUSSION THREAD

**Discussion by wukongman** [gkd-kit/gkd#837] [Q&A] [answered]
软件体验相当不错，当我给朋友推荐时，因为软件图标和通知图标为蔡徐坤，朋友经常问我是不是粉丝，蔡徐坤作为一个未成年性情的劣质明星，我既不是黑粉也不是粉丝，这对我带来了许多困扰，希望作者能够设计更加反应产品特色的图标替代现有的蔡徐坤元素

**Comment by lisonge** [comment]
> 蔡徐坤作为一个未成年性情的劣质明星

虽然但是，这是凡凡，不是坤坤

服了能不能先搞清楚，别再摸黑我们坤坤

**Reply by dearmb** [reply]
是凡凡吗？不是打篮球那个坤坤吗？我一直以为是坤坤，哈哈哈

**Reply by wukongman** [reply]
抱歉，这些明星的事情不太清楚，如果哪里说错了，还希望粉丝原谅，我只是希望大家玩梗适度，很多人并不关心娱乐圈，而软件传播过程中的明星元素对使用者带来了困扰，希望开发者能够适当调整软件图标

**Reply by lisonge** [reply, marked as answer]
你可以自己编译修改图标

遵守修改规范即可 <https://github.com/orgs/gkd-kit/discussions/679>

> 而软件传播过程中的明星元素对使用者带来了困扰

你要求去掉坤坤的元素，这何尝不是对坤坤粉丝使用者的一种困扰呢？

**Reply by wukongman** [reply]
好吧，你的软件你做主，你有不接受建议的权力，我可能会寻找软件替代品

**Comment by cvpv** [comment]
你们真的是蔡徐坤后援团ikun吗？还是玩梗的，说真的，这个梗玩了很多年，但是我还是很喜欢，但我不是蔡徐坤的粉丝。只是坤坤迷因文化的粉丝。

**Reply by AIsouler** [reply]
> 这个梗玩了很多年

厉不厉害你鸡哥🤣🤣🤣

**Comment by ccav14** [comment]
软件应该改名CXK

**Comment by Snowwhite-cute** [comment]
不知不觉，GKD已经诞生了两年半了

**Comment by VolcanoSAhrimp** [comment]
这是我用过最方便和自由的跳广告的app了

**Comment by Luoyacheng** [comment]
不了解梗的人也无法通过图标看出来。
不过我也希望作者能提供多图标选择，能有一个和软件作用相关的图标。
